src/main/java/com/infsis/socialpagebackend/dtos/FollowerDTO.java

```diff
@@ -0,0 +1,12 @@
+package com.infsis.socialpagebackend.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class FollowerDTO {
+    private String name;
+    private String las_name;
```

Comment:
it should be **last_name**, please fix it

Reply:
xd